data/scenarios/Challenges/Ranching/gated-paddock.yaml

```diff
@@ -0,0 +1,161 @@
+version: 1
+name: Pastoral Island
```

Comment:
I'm sure you have already spent many, many hours on this, so you should definitely add an `author: ` field with your name! :smile: